CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 * Added support for yml hidden parameters for `xsoar_saas` marketplace, as part of the **prepare_content** command.
 * Added support for custom documentation that will appear only in `xsoar_saas` marketplace, as part of the **prepare_content** command.
 * Modified **prepare_content** command to be platform specific. For xsoar-saas and XSIAM regarding pack readme and integration description images in markdown files.
+* Fixed an issue where logging arguments were not in the standard kebab-case. The new arguments are: **console-log-threshold**, **file-log-threshold**, **log-file-path**.
```

Comment:
please pull from master and make sure this is in the right section

Reply:
https://github.com/demisto/demisto-sdk/pull/3529/commits/51a4a38e54f14288cc18857c20f9027a36d91c76